src/server.rs

```diff
@@ -57,6 +56,63 @@ impl Handler {
     }
 }
 
+#[cfg(feature = "secure")]
+mod imp {
+    use grpc_sys::{self, GrpcServer};
+
+    use credentials::ServerCredentials;
+
+    pub struct Binder {
+        pub host: String,
+        pub port: u16,
+        cred: Option<ServerCredentials>,
```

Comment:
How about field attributes?

Reply:
Only the struct is unified, I still need to write several implementations for each feature respectively, which make things complicated. Use module is more simple and clear.

Reply:
LGTM